_applications/equoid.adoc

```diff
@@ -0,0 +1,223 @@
+
+
+==== Data Handler
+
+`WINDOW_SECONDS` - Size, in seconds of window for which the top-k elements should be determined. 
+
+`SLIDE_SECONDS` - Size, in seconds, of the amount to slide the sample window by each iteration. 
+
+`BATCH_SECONDS` - Size, in seconds, of the batch size to be acquired from the broker.
+
+`OP_MODE` - As with the data publisher, single for a list of field values from which random elements will be generated, dual for two lists of field values from which random elements will be generated, linear for a dataset meant to be read in sequence and transmitted.
+
+
+==== Settings for Examples
+
+Single-field sales messages generated from a set of possible values for item ID as provided by UCI dataset::
+* `DATA_URL_PRIMARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodes.txt for data publisher 
+* `OP_MODE` single for both data publisher and handler
+
+Dual-field sales messages generated from a set of possible values for item ID and country as provided by UCI dataset::
+* `DATA_URL_PRIMARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodes.txt for data publisher 
+* `DATA_URL_SECONDARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/Countries.txt for data publisher 
+* `OP_MODE` dual for both data publisher and handler
+
+Single-field sales messages sent in order as provided in the UCI dataset::
+* `DATA_URL_PRIMARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodesLinear.txt for data publisher 
+* `OP_MODE` linear for both data publisher and handler
+
+[[usage]]
+== Usage
+
+=== Web Interface
+With all the microservices running, you can now view a visualization of the data stream's top-ks for the parameters you have chosen. By browsing to the OpenShift web UI for your cluster, logging in, and looking at your project's overview, you should see a pod/application named equoid-ui with an associated URI. You could also run `oc get routes` from the command line to get the URI. By browsing to the address, you will then be at the initial splash screen for your equoid system, as shown in the following:
+
+pass:[<img src="/assets/equoid/Splash-Screen.png" alt="Alt text" class="img-responsive arch" width="800px">]
+
+By clicking on "sign in" you will be taken to the login screen. You can log in using either of the default credential sets listed on the login page, then click on the graph tab. You should see a visualization of your stream's data which updates periodically. Similar to the following, which shows an example of Equoid determining the top-3 items for 20 second windows. By hovering over areas of the graph, a breakdown of the items within received within the time window are displayed, as shown in the figure.  
```

Comment:
Similar to the following, which ....

--? Below, there is an example of Equoid determining the top-3 .... 

?

Reply:
Changed to "Similar to the following figure", thanks